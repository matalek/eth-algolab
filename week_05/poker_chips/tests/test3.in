10
4
11 12 157 1
27 36 79 74 51 58 18 95 5 29 100
98 80 88 1 13 96 24 60 56 47 77 87
65 13 96 74 72 50 51 66 21 58 98 75 49 97 59 98 15 37 84 16 55 93 3 49 18 2 96 59 85 59 97 49 43 27 19 99 79 40 70 84 64 15 46 1 98 27 64 78 40 7 64 44 93 5 56 24 88 70 83 92 17 16 96 78 33 71 33 11 34 26 17 80 90 77 37 90 20 45 61 46 76 66 15 32 72 1 81 15 86 18 23 86 26 29 26 26 38 80 24 64 64 40 92 91 30 77 30 31 70 23 37 13 68 72 61 76 96 14 11 58 12 14 77 38 10 29 94 97 48 41 41 92 78 36 33 77 95 91 6 98 17 55 54 34 37 61 20 62 62 26 20 48 32 74 73 27 36
47
2
117 489
81 62 70 32 93 60 43 41 18 93 87 63 87 19 62 5 9 90 93 81 83 45 54 14 19 85 10 100 43 57 90 39 62 92 33 75 93 88 34 14 10 48 46 6 34 97 56 98 68 79 77 54 6 82 12 3 52 80 97 45 65 59 97 28 99 15 67 24 46 17 30 90 78 16 42 67 38 36 15 56 13 78 47 98 18 23 42 85 91 85 10 13 26 51 11 89 60 50 38 13 26 36 51 47 45 78 74 20 75 39 34 67 35 16 4 89 28
79 84 63 70 4 29 49 53 98 31 18 61 44 52 13 71 2 70 12 47 93 26 25 3 88 30 42 73 93 11 100 46 46 74 81 54 15 51 54 62 18 52 86 29 69 3 85 94 59 23 23 46 11 2 43 10 74 40 32 90 80 85 74 7 20 76 29 13 9 72 83 27 100 28 76 32 38 71 9 91 100 24 66 75 44 19 7 51 88 49 30 50 36 19 41 24 89 78 64 69 73 16 2 31 92 98 67 66 85 11 24 78 78 85 69 28 87 11 48 18 73 8 34 80 14 31 74 10 39 28 28 84 95 37 16 89 78 15 26 18 24 91 95 91 62 9 8 8 30 12 51 27 32 7 15 20 26 66 50 80 93 11 11 37 83 39 74 4 7 89 13 17 84 29 94 15 52 95 41 70 22 58 51 85 50 92 9 39 2 49 51 25 54 92 44 63 71 25 98 49 88 5 56 36 20 69 51 57 93 53 38 7 17 99 55 56 56 29 43 82 58 96 68 56 78 59 25 30 53 90 40 69 16 89 99 100 21 75 23 75 33 86 39 38 23 86 89 87 87 51 19 24 3 43 78 67 64 8 23 44 48 23 63 78 89 58 41 10 52 24 24 87 16 80 33 93 44 87 7 3 40 36 94 22 64 26 46 5 38 14 34 88 35 30 6 37 38 24 90 26 52 66 26 30 24 85 3 64 68 56 31 82 27 32 43 72 16 76 52 53 100 85 99 42 44 21 99 47 79 9 100 22 56 68 63 2 31 55 13 22 15 75 74 88 29 100 38 56 71 41 51 4 63 85 28 26 89 89 47 55 79 87 6 60 14 58 62 53 22 14 15 56 49 31 18 64 55 39 28 87 42 36 98 56 58 59 89 59 89 39 68 72 24 86 11 19 18 11 45 55 69 30 91 13 41 52 32 35 53 75 71 87 1 19 47 2 97 98 62 32 57 85 12 50 35 22 57 84 56 80 71 35 11 35 37 63 45 18 25 81 76 90 41 64 66 35 86 46 3 88 86 47 33 5 85 47 57 9 40 38 38 88 87 9 53 100 66 59 50 2 58 95 98 39 67 38 49 93 30 84 82 97 1 50 56 91 1 95 29
3
49 21 47
48 29 41 13 98 72 51 98 58 11 51 70 22 32 44 2 93 57 26 51 16 51 54 11 58 43 44 61 81 35 86 85 34 16 46 18 21 59 35 48 46 84 42 47 37 70 5 1 13
24 38 30 15 69 78 60 42 56 29 87 37 35 21 15 66 36 11 4 36 83
37 11 36 40 60 7 16 70 55 57 99 28 11 97 76 42 36 100 39 97 18 49 25 37 52 38 84 12 83 17 85 58 44 32 75 60 29 76 5 90 54 97 62 61 46 86 71
3
115 45 11
1 1 2 1 2 1 1 2 2 1 2 2 2 2 2 1 2 2 1 1 1 2 1 2 2 1 1 2 2 1 2 1 1 1 1 1 1 1 2 2 2 2 2 2 1 2 1 2 2 2 1 1 2 1 2 2 2 1 2 2 2 2 2 1 1 1 1 2 2 1 2 2 1 1 1 1 1 1 1 2 1 1 1 2 2 2 2 1 2 1 2 1 1 2 2 2 2 2 1 1 2 2 2 2 1 2 2 1 2 2 2 2 2 1 1
2 2 2 1 2 1 2 1 2 2 2 2 2 1 1 1 1 2 1 2 1 2 1 2 1 2 1 2 1 1 2 1 2 1 2 1 1 2 2 1 1 1 2 2 1
1 1 2 1 2 1 2 2 2 2 1
3
113 12 28
1 2 2 2 1 1 1 1 1 2 1 1 2 1 2 1 1 2 2 1 2 1 2 1 1 1 1 1 2 1 2 1 2 1 2 1 1 1 2 2 1 2 2 2 2 1 2 1 1 2 1 1 2 2 1 2 2 2 1 1 2 1 2 1 2 1 2 1 2 2 2 1 2 2 1 2 2 2 2 1 1 2 2 2 2 1 1 2 2 2 2 2 1 1 1 2 2 1 2 1 2 2 2 2 1 1 1 1 1 2 2 1 2
1 2 2 1 2 2 1 1 1 1 2 2
1 1 2 2 1 2 2 2 2 1 1 1 2 2 2 1 1 1 1 1 2 1 1 1 2 2 2 2
2
268 178
2 2 1 1 2 2 1 2 2 1 1 2 1 1 1 2 1 2 1 1 2 2 1 1 1 1 2 1 2 2 1 2 1 2 2 1 2 2 2 1 1 1 1 1 1 2 2 1 2 1 1 2 1 2 2 1 2 2 1 1 2 1 1 2 2 1 2 1 1 1 2 2 2 1 2 1 2 1 2 1 2 1 1 2 2 1 2 1 2 1 1 1 1 1 1 1 2 2 2 2 1 1 2 1 1 1 1 1 2 2 2 1 2 1 1 1 2 2 1 2 1 1 1 1 2 1 1 2 2 1 1 1 1 2 1 2 2 1 1 1 1 1 2 1 2 1 2 2 2 2 2 1 2 2 2 1 2 1 1 2 2 2 2 2 2 1 1 1 1 1 2 1 1 2 2 2 2 1 2 2 1 2 1 2 1 1 2 1 2 1 1 2 1 1 1 2 2 2 1 1 1 2 1 1 1 2 1 1 1 2 2 2 2 1 2 1 2 1 1 2 2 2 1 2 1 1 1 2 1 2 1 1 1 2 1 2 1 2 2 1 1 1 2 2 1 2 2 2 1 1 2 1 1 1 2 1 1 1 2 1 2 1 1 1 2 1 2 2
2 2 1 2 1 2 2 2 1 1 2 2 1 2 2 2 2 2 2 2 1 1 2 2 2 1 2 2 1 2 1 1 1 1 1 1 2 1 2 2 2 1 1 2 2 2 2 1 1 2 2 1 2 2 2 1 2 1 1 1 1 1 2 2 2 2 2 2 1 1 2 1 1 2 2 1 2 1 2 2 1 2 1 1 2 1 1 2 1 2 1 2 2 2 1 1 1 1 2 1 2 2 2 2 2 2 1 1 2 2 2 1 2 2 1 1 2 2 1 1 1 2 2 1 1 1 1 2 1 1 2 1 1 1 1 1 2 1 1 1 2 1 1 2 2 2 2 2 2 2 2 1 2 1 1 2 2 2 2 2 2 1 2 2 1 2 2 1 2 1 2 1 1 2 2 1 1 1
3
10 78 48
34 62 62 9 22 48 57 15 29 55
34 62 62 9 22 48 57 15 29 55 42 31 77 22 16 15 32 36 56 48 25 72 33 55 29 22 15 51 16 10 74 37 34 75 74 73 39 36 69 72 44 37 30 20 68 53 45 62 36 46 52 25 31 30 62 19 53 70 40 42 17 54 50 47 16 28 71 9 63 56 75 11 31 54 8 25 44 50
45 72 53 25 44 36 31 53 39 50 70 25 44 16 77 74 34 68 25 36 42 48 56 33 74 54 29 15 9 69 46 72 37 75 36 22 55 56 16 32 71 15 10 34 9 40 50 8
3
125 39 5
113 113 55 125 113 119 70 65 100 61 88 47 106 9 29 29 6 30 118 81 95 108 78 101 104 104 9 69 4 110 3 115 18 89 91 47 103 9 34 13 4 19 84 16 91 44 35 72 66 1 75 123 81 22 48 57 55 110 91 74 51 116 54 9 92 112 65 106 112 115 89 73 27 72 41 17 118 88 13 65 65 56 20 114 52 39 94 30 88 8 99 3 37 101 58 124 34 86 17 61 115 43 9 66 6 16 74 69 26 5 76 88 60 85 52 93 84 119 27 116 34 122 102 17 120
113 113 55 125 113 119 70 65 100 61 88 47 106 9 29 29 6 30 118 81 95 108 78 101 104 104 9 69 4 110 3 115 18 89 91 47 103 9 34
35 95 9 114 74
2
155 370
9 335 316 230 69 89 282 271 272 190 238 41 228 274 212 238 187 228 283 217 238 62 353 150 197 214 309 313 310 272 62 231 368 183 135 239 290 153 186 176 136 46 231 358 174 171 368 124 121 113 353 292 4 250 51 5 157 217 245 12 81 62 312 342 292 112 186 213 356 19 266 250 286 222 321 145 314 1 104 23 361 225 180 213 3 89 115 140 109 5 227 147 314 248 171 88 123 98 216 154 115 93 311 322 147 7 278 124 108 348 262 31 68 130 177 229 193 219 211 225 94 81 343 5 179 251 63 147 306 205 195 206 261 316 297 253 348 253 257 105 223 95 82 149 101 168 138 317 288 30 224 233 116 267 66
113 341 176 189 243 338 43 3 313 342 92 335 324 331 317 342 135 253 195 65 68 240 248 135 265 312 278 227 267 217 9 297 317 337 202 368 208 37 257 145 2 363 229 233 186 101 197 178 107 347 50 202 271 57 141 82 316 128 346 114 272 11 213 231 195 16 189 150 224 149 18 5 154 75 282 358 216 347 274 239 6 260 193 117 201 136 314 314 356 183 154 113 15 154 171 317 321 28 135 283 230 98 353 310 222 213 27 72 69 108 110 94 203 81 312 60 277 85 156 264 238 141 288 221 238 104 49 360 353 192 266 223 322 337 58 321 18 204 235 63 326 112 292 157 311 109 29 228 251 311 228 250 207 348 242 116 56 348 238 62 280 364 259 220 162 22 368 343 13 5 245 194 168 361 45 121 70 123 250 361 197 124 213 210 304 271 115 187 289 145 140 112 63 306 19 157 124 271 342 1 349 211 231 261 40 95 358 292 311 114 124 219 344 323 215 115 14 295 287 176 225 272 190 366 116 210 267 68 126 218 275 312 324 361 192 147 134 89 212 206 50 90 66 268 339 204 193 352 174 211 299 51 4 4 171 142 318 71 46 307 354 306 201 336 296 217 67 214 316 287 15 268 186 7 264 30 354 301 253 1 360 41 22 319 12 115 21 35 90 304 113 332 147 179 227 128 70 46 226 177 151 205 235 309 198 335 361 130 213 19 313 338 286 45 338 105 168 159 62 18 138 303 88 93 289 21 290 89 349 174 34 159 225 91 62 22 353 113 309 81 34 266 135 153 262 119 125 228 5 147 238 74 180 174 20 368 286 184 347 348 31 263 367 314 272 23 340 102 129 28
4
144 21 1 4
90 86 59 142 129 137 125 9 65 65 142 135 111 135 88 90 118 100 70 44 78 101 53 121 40 9 94 51 84 51 33 33 35 33 34 31 110 137 63 9 94 103 92 140 51 72 37 28 39 12 129 68 136 74 88 77 66 19 99 93 9 112 50 112 32 118 103 104 122 126 68 100 76 85 78 109 99 71 43 38 62 32 47 144 7 108 102 86 82 118 56 17 34 101 43 54 36 25 82 46 44 144 70 59 94 125 144 61 27 69 67 74 67 101 12 117 56 44 66 134 84 115 112 136 102 21 130 90 2 54 114 84 62 32 33 82 139 87 96 39 99 69 37 62
90 86 59 142 129 137 125 9 65 65 142 135 111 135 88 90 118 100 70 44 78
90
90 86 59 142
