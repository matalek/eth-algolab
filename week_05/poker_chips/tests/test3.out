13
36
14
56
40
172
25
43
29
27
